server.js

```diff
@@ -733,7 +707,7 @@ app.post('/questionmod', function(req, res) {
     var qid = parseInt(req.body.id);
     var q = req.body.question;
 
-    questions.updateQuestionByIdWithRedirection(qid, q, function(err, result) {
+    questions.updateQuestionById(qid, q, function(err, result) {
```

Comment:
This change is the same as in PR #36, dissolve one or the other?

Reply:
I think I branched of the wrong branch. Will fix this asap